includes/forum.php

```diff
@@ -437,6 +437,10 @@ function prepare() {
                     $this->current_view = 'overview';
                 }
             break;
+	        case apply_filters('asgarosforum_custom_page_'.$this->current_view.'_switch','custom'):
+	        	// Allow for custom view
```

Comment:
We should put this check in the default-condition because otherwise we always have a view called `custom` available which is not used by default. By default the value filtered can be `false`, otherwise register a view with the filtered string.